packages/forms-web-app/src/dynamic-forms/question.js

```diff
@@ -15,66 +19,36 @@ const nunjucks = require('nunjucks');
  * @class
  */
 class Question {
-	/**
-	 * @type {string} - title used in the summary list
-	 */
+	/** @type {string} - html page title, defaults to question if not provided */
+	pageTitle;
+	/** @type {string} - title used in the summary list */
 	title;
-	/**
-	 * @type {string} - question shown to user on question page
-	 */
+	/** @type {string} - question shown to user on question page */
 	question;
-	/**
-	 * @type {string} - additional information to user about the question
-	 */
+	/** @type {string|undefined} - additional information to user about the question */
 	description;
-	/**
-	 * @type {string} the folder name of the view
-	 */
+	/** @type {string} the folder name of the view */
 	viewFolder;
-	/**
-	 * @type {string} the unique name of the input on the page, also used as a url segment (should this be separated)
-	 */
+	/** @type {string} the unique name of the input on the page, also used as a url segment (should this be separated) */
 	fieldName;
-
-	/**
-	 * @type {boolean} if the question should appear in the journey overview task list or not
-	 */
+	/** @type {boolean} if the question should appear in the journey overview task list or not */
 	taskList;
-
-	/**
-	 * @type {string} alt text for displaying in tasklist
-	 */
+	/** @type {string} alt text for displaying in tasklist */
 	altText;
-
-	/**
-	 * @type {function} function that customises the formatting of the question
-	 */
+	/** @type {function} function that customises the formatting of the question */
 	format;
-
-	/**
-	 * @type {function} function providing a custom rendering action
-	 */
+	/** @type {function} function providing a custom rendering action */
 	renderAction;
-
-	/**
-	 * @type {function} function providing a custom saving action
-	 */
+	/** @type {function} function providing a custom saving action */
 	saveAction;
-
-	/**
-	 * @type {Array} array of validators that a question uses to validate answers
-	 */
+	/** @type {Array.<BaseValidator>} array of validators that a question uses to validate answers */
 	validators = [];
-
-	/**
-	 * @type {string} hint text displayed to user
-	 */
+	/** @type {string} hint text displayed to user */
 	hint;
-
-	/**
-	 * show return to listing page link after question
-	 */
+	/** @type {boolean} show return to listing page link after question */
 	showBackToListLink = true;
+	/** @type {string|undefined} alternative url slug */
+	url;
 
 	bulletPoints = [];
```

Comment:
We can get rid of this as per other comment. I know it's not strictly part of this PR but may as well clean it up whilst we're at it if possible please.